draft-setty-cfrg-spartan-incubation.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "TODO - Your title"
-abbrev: "TODO - Abbreviation"
+title: "Spartan High-speed zkSNARKs (incubation)"
```

Comment:
Do we want the title to be a bit broad? For example SNARK-based verifiable credentials?

Reply:
This is just a starting point, we'll definitely be updating the title in the future including renaming the draft itself.